src/StoreKeeper/WooCommerce/B2C/FileExport/CategoryFileExport.php

```diff
@@ -127,4 +128,57 @@ private function getParentSlug(array $map, int $parentId = 0)
 
         return $parentSlug;
     }
+
+    private function sortCategoriesByParent(array $mappedCategories): array
+    {
+        $categories = [];
+
+        foreach ($mappedCategories as $term) {
```

Comment:
maybe better to use [usort](https://www.php.net/manual/en/function.usort.php)? It pretty hard for me to verify if that one works with the splicing and adding categories.

Reply:
Updated to do category tree and use usort instead.